workos/sso.py

```diff
@@ -50,7 +50,8 @@ def get_authorization_url(
             str: URL to redirect a User to to begin the OAuth workflow with WorkOS
         """
         params = {
-            "client_id": workos.project_id,
+            "client_id": workos.client_id,
+            "project_id": workos.project_id,
```

Comment:
Can we raise a warning if `project_id` is used here? It can be similar to the call made [here](https://github.com/workos-inc/workos-python/blob/master/workos/sso.py#L118-L121).

Reply:
We don't need to pass `project_id` in the parameters here (an example call can be seen in the docs [here](https://workos.com/docs/reference/sso/redirect-uri/get)). `workos.project_id` or `workos.client_id` should be assigned to `params['client_id']`, but if `workos.project_id` is non-null, we should raise the warning.